### ISSUE TITLE
borderRadius is not working with this plugin.

### ISSUE THREAD

**Issue by alinz** [chirag04/react-native-dashed-border#2]
Is it possible to use borderRadius here? I can't seem to get it working. Any suggestion?

Here's the Sample code:

``` js
import React, { Component, StyleSheet, View, Text } from 'react-native';
import DashedBorder from 'react-native-dashed-border';

const styles = StyleSheet.create({
  container: {
    height: 100,
    width: 100,
    borderRadius: 50,
    borderWidth: 1,
    borderColor: '#56AEF5'
  },
  label: {
    textAlign: 'center',
    color: '#56AEF5',
    justifyContent: 'center'
  }
});

export default class AddPhoto extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <DashedBorder
        color={'#56AEF5'}
        lineDashPattern={[4,2]}
        style={styles.container}>
        <Text style={styles.label}>Add Photo</Text>
      </DashedBorder>
    );
  }
}
```

it produces the following images:

![screen shot 2015-07-27 at 9 41 08 am](https://cloud.githubusercontent.com/assets/967050/8907374/7c37017e-3443-11e5-91ee-c1a24458a896.png)

I want to have circle-dashed look!

**Comment by chirag04** [comment]
I actually never tried borderRadius with this plugin but it would be interesting to support it if it's not working. 

I expected it to work out of the box since `DashedBorder` is just a view and should support all the styles. I will try to debug later when i have some time. Feel free to investigate and PR.

**Comment by chirag04** [comment]
fixed in v0.1.6.

Thanks @alinz
